packages/story-editor/src/components/canvas/frameElement.js

```diff
@@ -269,6 +283,18 @@ function FrameElement({ id }) {
     }
   }, [setFocusGroupCleanup, isSelected]);
 
+  // translators: %s: Name of element
+  const lockedElementText = __('Locked element: %s', 'web-stories');
+
+  // translators: %s: Name of element
+  const unlockedElementText = __('Element: %s', 'web-stories');
+
+  const layerText = getLayerText(element);
+  const elementText = element.isLocked
+    ? lockedElementText
+    : unlockedElementText;
+  const elementLabel = sprintf(elementText, layerText);
```

Comment:
To address the i18n warning:

```suggestion
  const layerText = getLayerText(element);
  const elementLabel = element.isLocked
    ? sprintf(
        // translators: %s: Name of element
        __('Locked element: %s', 'web-stories'),
        layerText
      )
    : sprintf(
        // translators: %s: Name of element
        __('Element: %s', 'web-stories'),
        layerText
      );
```

Reply:
Thanks!